python/mxnet/ndarray/ndarray.py

```diff
@@ -157,6 +157,11 @@ def waitall():
     """Wait for all async operations to finish in MXNet.
 
     This function is used for benchmarking only.
```

Comment:
Not many people will understand what Rethrowing exception means. I suggest the below:

. warning:: This API is for internal purpose  for benchmarking) , it does not do any error handling and can cause silent failures. Use it only if you are confident that your code error free, suggested to call `wait_to_read` on all outputs after `waitall`.